src/core/utils/history_tracker.cpp

```diff
@@ -74,7 +74,7 @@ void Local::RecordNetworkInfo(void)
     return;
 }
 
-void Local::RecordMessage(const Message &aMessage, const Mac::Address &aMacAddress, MessageType aType)
+void Local::RecordMessage(const Message &aMessage, const Mac::Address &aMacAddress, MessageType aType, bool isTxSuccess)
```

Comment:
Rename `isTxSuccess -> aIsTxSuccess` (to follow style guide and naming convention for args).